setup.cfg

```diff
@@ -20,6 +23,9 @@ keywords =
     varlink
     libpod
     podman
```

Comment:
I think these keywords are producing an invalid PKG-INFO file, they look like this on Travis:

```
Keywords: varlink
libpod
podman
```

But should look like:

```
Keywords: varlink,libpod,podman
```

However I'm not sure where the bug lies.

Reply:
> I think these keywords are producing an invalid PKG-INFO file, they look like this on Travis:
> 
> ```
> Keywords: varlink
> libpod
> podman
> ```
> 
> But should look like:
> 
> ```
> Keywords: varlink,libpod,podman
> ```
> 
> However I'm not sure where the bug lies.

@di Oh thanks, good catch!

[I have already fixed this error on `pbr`](https://github.com/openstack/pbr/commit/2e3aa0d8519a35a6935aadb2e3322c58f9367fa1) but I need to verify why it's occur here.

I'll test with the inline syntaxe `keywords: varlink, libpod, podman` but I don't think it's an issue who can explain why `twine check dist/*` fail on travis.